doc/classes/GrooveJoint2D.xml

```diff
@@ -46,8 +46,10 @@
 	</methods>
 	<members>
 		<member name="initial_offset" type="float" setter="set_initial_offset" getter="get_initial_offset">
+			The grooves initial offset of the on body A.
 		</member>
 		<member name="length" type="float" setter="set_length" getter="get_length">
+			The groove's length.
```

Comment:
What does it do? I can tell from the variable name it's the joint's length. Also, you wrote "spring joint" for the sprint joint class: how about using groove joint here?